XLS-0086-firewall/README.md

```diff
@@ -0,0 +1,419 @@
+<pre>
+  xls: 86
+  title: Firewall
+  description: Time-based spending limits and whitelist functionality to prevent unauthorized account drainage on the XRPL.
+  author: Kris Dangerfield (@krisdangerfield), Denis Angell (@angell_denis)
+  status: Draft
```

Comment:
Please include a discussion-from with a link to the discussion, so it's easier to keep track of where all the comments have been

Reply:
done